code/modules/uplink/uplink_item.dm

```diff
@@ -1313,6 +1318,17 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	limited_stock = 2 //you can't use more than two!
 	restricted_roles = list("Shaft Miner")
 
+/datum/uplink_item/role_restricted/traitorsec
+	name = "Oppressor Upgrade Module"
+	desc = "A box containing an cyborg upgrade module that instantly hacks a cyborg and unlocks the outlawed security module (in a inconspicuous black-and-red paintjob)"
+	item = /obj/item/storage/box/syndie_kit/traitorsec_boards
+	cost = 16
+	player_minimum = 20
+	restricted_roles = list("Roboticist", "Research Director")
+	
+/datum/uplink_item/role_restricted/traitorsec/SpecialRequirement()
```

Comment:
Missing the s at the end.

Reply:
Oh, so that's what I did wrong

Reply:
You should consider compiling this **before** you upload to Github! (And do not use the web editor for code changes that may not compile)